client/packages/common/src/hooks/useUrlQuery/useUrlQueryParams.ts

```diff
@@ -134,16 +129,21 @@ export const useUrlQueryParams = ({
     onClearFilterRule: key => updateFilterQuery(key, ''),
     filterBy: getFilterBy(),
   };
+
+  const pageSize =
+    urlQuery['pageSize'] && typeof urlQuery['pageSize'] === 'number'
+      ? urlQuery['pageSize']
+      : DEFAULT_RECORDS_PER_PAGE;
```

Comment:
storedRowsPerPage definitely was working before, just tested on develop. I'd default to that in here where you've been using DEFAULT_RECORDS_PER_PAGE... we don't want to break ability to change page size in the interim